en/docs/tutorials/create-rest-api-component.md

```diff
@@ -0,0 +1,299 @@
+
+        Click the last **+** icon in the low-code diagram and click **Variable**. Then enter information as follows:
+
+         | **Field**      | **Value**                     |
+         |----------------|-------------------------------|
+         | **Type**       | `json`                        |
+         | **Name**       | `payload`                     |
+         | **Expression** | `{country : country, totalCasesPerMillion : totalCasesPerMillion}` |
+
+    2. Click **Save**.
+
+    3. To respond with the JSON payload, add a `Return` statement.
+
+        Click the last **+** icon in the low-code diagram and click **Return**.
+
+        In the **Return Expression** field, enter `payload`.
+
+        Save the information.
+
+Now you have completed designing the `CovidStatus` service.
+
+The low-code diagram looks as follows:
+
+ ![Add service construct](../assets/img/tutorials/rest-api/low-code-diagram.png){.cInlineImage-full}
+
+The code view looks as follows:
+
+```ballerina
+import ballerinax/worldbank;
+import ballerinax/covid19;
+import ballerina/http;
+
+service /CovidStats on new http:Listener(9090) {
+    resource function get stats/[string country]() returns json|error {
+        covid19:Client covid19Client = check new ({});
+        covid19:CovidCountry statusByCountry = check covid19Client->getStatusByCountry(country);
+        int totalCases = <int>statusByCountry.cases;
+        worldbank:Client worldbankClient = check new ({});
+        worldbank:IndicatorInformation[] populationByCountry = check worldbankClient->getPopulationByCountry(country);
+        int populationMillions = (populationByCountry[0]?.value ?: 0) / 1000000;
+        decimal totalCasesPerMillion = <decimal>(totalCases / populationMillions);
+        json payload = {country: country, totalCasesPerMillion: totalCasesPerMillion};
+        return payload;
+    }
+}
+```
+Now you can run the REST API and test it to see whether it works as expected.
+
+### Run and test the REST API
+
+To run the API, click **Run** (above the low-code diagram).
+
+Once the REST API is successfully started, the following appears in the terminal log.
+
+```
+Running executable
+```
+Now you can try out the REST API as follows:
+
+1. Click **Try it** As a result, a test view opens to the right of the page.
+
+2. In the test view, expand the **GET** resource.
+
+3. Click **Try it out**.
+
+4. In the **country** field, enter `USA`.
+
+5. Click **Execute**.
+
+The following is displayed as the response body under **Responses**.
+
+![Try-out response](../assets/img/tutorials/rest-api/try-out-response.png){.cInlineImage-half}
+
+The REST API you created works as expected. Therefore, now you can commit it.
+
+### Commit the REST API to GitHub
+
+The REST API you designed is currently available only in the VS Code Editor. To use it, you need to save it in the Choreo Console. You can do this by committing the REST API configuration into a private repository in GitHub as follows:
```

Comment:
We need to also  mention that Choreo maintains code in private GitHub repos. Let's point to the concept doc to learn more.